src/server/handlers/create.rs

```diff
@@ -22,14 +32,49 @@ pub struct CreateSubnetResponse {
 }
 
 /// The create subnet json rpc method handler.
-pub(crate) struct CreateSubnetHandler {}
+pub(crate) struct CreateSubnetHandler<T: JsonRpcClient> {
```

Comment:
We may want this to be a `SubnetHandler` that aggregates all subnet manager methods that way we only have to pass the manager pool once.